Directory.Packages.props

```diff
@@ -15,6 +15,8 @@
     <PackageVersion Include="MongoDB.Driver" Version="2.10.4" />
     <PackageVersion Include="Moq" Version="4.14.5" />
     <PackageVersion Include="Portable.BouncyCastle" Version="1.8.6.7" />
+    <PackageVersion Include="Quartz.Extensions.DependencyInjection" Version="3.2.0-preview-20200806-1639" />
+    <PackageVersion Include="Quartz.Extensions.Hosting" Version="3.2.0-preview-20200806-1639" />
```

Comment:
I'm trying to get [Quartz.Extensions.Hosting](https://www.nuget.org/packages/Quartz.Extensions.Hosting/) to Quartz project, until then need to use the preview feed. Fortunately Quartz is not protected on NuGet and people cannot use the prefix without being in Quartz.NET organization, less things to worry...

Reply:
Good to know, thanks! We've had similar issues with a few aspnet-contrib packages, but luckily, all the libs authors we've spoken to all have been kind enough to transfer the ownership to the aspnet-contrib org 😃

Reply:
Yes the two other integration package names were also taken, but fortunately there was no problem for the authors to contribute the package id.